lib/services/microsoft_teams.rb

```diff
@@ -0,0 +1,103 @@
+class AhaServices::MicrosoftTeams < AhaService
+  caption do |workspace_type|
+    subject =
+      case workspace_type
+      when "multi_workspace" then "product or workspace"
+      when "product_workspace" then "product"
+      when "marketing_workspace" then "workspace"
```

Comment:
might want to else this. it can be easily overlooked when adding a new workspace.